app/commands.go

```diff
@@ -1,96 +0,0 @@
-package app
```

Comment:
why are these files removed?

Reply:
They duplicated the code in conode.go which is used now instead.